app/src/debug/kotlin/io/homeassistant/companion/android/developer/DevPlaygroundActivity.kt

```diff
@@ -69,6 +70,11 @@ private fun DevPlayGroundScreen(context: Context? = null) {
         }) {
             Text("Start Settings")
         }
+        Button(modifier = Modifier.padding(top = 16.dp), onClick = {
+            context?.run { startActivity(BarcodeScannerActivity.newInstance(this, 0, "Title", "Subtitle", null)) }
```

Comment:
Unrelated to edge to edge design: to better match the current frontend usage of the scanner, I would set the action so it is displayed.

```suggestion
            context?.run { startActivity(BarcodeScannerActivity.newInstance(this, 0, "Title", "Subtitle", "Action")) }
```